hugo/content/core/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Core"
+date: 2023-06-26T09:36:16-04:00
+draft: true
+---
+
+# DORA Core
+DORA’s research program is continuous and ongoing; each year brings new avenues of inquiry, and each analysis yields new insights. At the cutting edge, new concepts are introduced frequently, and artifacts (especially the [State of DevOps Reports]({{< relref "/publications" >}})) are continuously released, revealing new insights and dynamics. With each study, some prior findings are reinforced, while others may be called into question. This is a hallmark of good science: any new finding is to be considered suspect—intriguing, but suspect—until it has been validated through replication and application. Meanwhile, practitioners are encouraged to apply the research in their own professional contexts. In such contexts, it can be challenging to keep up with the latest developments from DORA: cultural transformation efforts in a large organization require a steady hand, persistent over stretches of time measured across multiple years. An attempt to update one’s practices to match the pace of the research can lead to counterproductive churn. In these contexts, it’s more practical to rely on “evergreen” artifacts. DORA Core represents a distillation of DORA’s most foundational findings: metrics, capabilities, and outcomes that the research has repeatedly surfaced. It enables teams to focus their improvement efforts even more precisely on what is likely to produce tangible benefits to their organizational goals and quality of life. 
+
+<img src="dora-core-model.svg" style="width:100%">
+
+> [Download as PDF](dora-core-model.pdf)
+
+## FAQ
+#### What’s in Core? What isn’t?
+DORA Core encompasses the set of entities that have been perennially studied by DORA and whose interactions are well supported. These entities include capabilities like Continuous Delivery and Generative Culture, metrics like the “Four Key'' metrics, and outcomes like organizational performance. These established ideas contrast with areas of inquiry, or divergent predictive pathways, which have shown promise when we’ve studied them, but aren’t (yet) as well supported as part of DORA’s predictive model. An entity may be considered for Core if: a) it has been researched at least twice; b) practitioners have found it valuable; c) continued exploration is warranted. This rubric is, by design, a subjective one, which draws heavily on the data and analyses, but it is ultimately guided by intuition. So please contribute! Your intuition is valuable; if you feel like something doesn’t accurately reflect the research, please let us know.
```

Comment:
Should we link to the capabilities when mentioning them? 

i.e., "...like [Continuous Delivery](/devops-capabilities/technical/continuous-delivery/) and [Generative Culture](/devops-capabilities/cultural/generative-organizational-culture/)..."

Reply:
good call -- fixed